branding/css/susemanager-theme.less

```diff
@@ -1302,3 +1302,9 @@ input:disabled+label {
   color: @spacewalk-gray;
 }
 
+select.small-select {
+  height: 25px;
```

Comment:
Do we still need pixels in CSS? :confused:

Reply:
Generally speaking, I am afraid we will never get rid of pixels, and sometimes it still makes sense btw. But in this case, you are totally right! Fixed now, thank you. :wink:

Reply:
@ncounter Do we have any rule on SUMA when to use pixels/em/rem? Anyway I think the important part here is to keep the consistency through out the site.
https://mindtheshift.wordpress.com/2015/04/02/r-i-p-rem-viva-css-reference-pixel/ this blogpost that mostly promote the use of pixels is really interesting.